packages/cf-style-provider/src/createRenderer.js

```diff
@@ -19,7 +19,8 @@ const mediaQueries = {
   mobileWide: `@media (min-width: ${variables.breakpoints.mobileWide})`,
   tablet: `@media (min-width: ${variables.breakpoints.tablet})`,
   desktop: `@media (min-width: ${variables.breakpoints.desktop})`,
-  desktopLarge: `@media (min-width: ${variables.breakpoints.desktopLarge})`
+  desktopLarge: `@media (min-width: ${variables.breakpoints.desktopLarge})`,
+  hover: `@media (hover)`
```

Comment:
Are we relying on this somewhere or is this just a enhancement? It doesn't seem to [be supported in Firefox](https://caniuse.com/#feat=css-media-interaction).